deployment/docker/alerts.yml

```diff
@@ -1,60 +1,7 @@
-# File contains default list of alerts for vm-single and vmagent services.
+# File contains default list of alerts for VictoriaMetrics single server.
 # The alerts below are just recommendations and may require some updates
 # and threshold calibration according to every specific setup.
```

Comment:
> this will be useful once alerts.yaml will be merged into cluster branch

what do you mean by this? The intention was just to keep a placeholder in cluster branch

Reply:
ahh. ok in this case fine. i thought that alerts will be copied over from cluster master to cluster branch 👍